job_templates/packaging_job.xml.em

```diff
@@ -134,7 +134,7 @@ if [ -n "${CI_AMENT_TEST_ARGS+x}" ]; then
   if [[ "$CI_AMENT_TEST_ARGS" != *-- ]]; then
     CI_AMENT_TEST_ARGS="$CI_AMENT_TEST_ARGS --"
   fi
-  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS"
+  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS --"
```

Comment:
If you look closely at that console output you see:
```
13:04:28 + export CI_ARGS=--do-venv --force-ansi-color --fastrtps --test-bridge --cmake-build-type RelWithDebInfo
13:04:28 + [ -n x ]
13:04:28 + [[ --retest-until-pass 5 != *-- ]]
13:04:28 /tmp/hudson3577370792795485862.sh: 24: /tmp/hudson3577370792795485862.sh: [[: not found
13:04:28 + export CI_ARGS=--do-venv --force-ansi-color --fastrtps --test-bridge --cmake-build-type RelWithDebInfo --ament-test-args --retest-until-pass 5
13:04:28 + export CI_ARGS=--do-venv --force-ansi-color --fastrtps --test-bridge --cmake-build-type RelWithDebInfo --ament-test-args --retest-until-pass 5 --ros1-path /opt/ros/kinetic
```

It was never appending `--` because of an error with the `[[` syntax (it's using dash).

I've opened https://github.com/ros2/ci/pull/26